### PR TITLE
dSPAC Support

### PR DESCRIPTION
**Pull request by ImNotOssy**
working now

no longer need to manually install packages, just did pip install -r requirements.txt

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -15,6 +15,11 @@ HEADLESS="true"
 # at the same brokerage with a comma, then separate account credentials with a colon
 # BROKER=BROKER_USERNAME:BROKER_PASSWORD,OTHER_BROKER_USERNAME:OTHER_BROKER_PASSWORD
 
+# dSPAC Pro
+# USE_EMAIL takes either TRUE or FALSE. TRUE mean email for username, FALSE means phone number
+# DSPAC=DSPAC_USERNAME:DSPAC_PASSWORD:USE_EMAIL TRUE/FALSE
+DSPAC=
+
```
Comment (MaxxRK) [inline .env.example:22]: As NelsonDane mentioned in BBAE I would remove the TRUE/FALSE if possible and just determine whether it's an email using a simple check such as '@'. Any option would work there.

**Review by MaxxRK — CHANGES_REQUESTED** [review]
This looks great for the most part. It now works for me. The comment above and below are the only changes needed that I see right now.

File: dspacAPI.py [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+        if DSPAC_EXTERNAL is None
+        else DSPAC_EXTERNAL.strip().split(",")
+    )
+    print("Logging in to DSPAC...")
+    for index, account in enumerate(DSPAC):
+        name = f"DSPAC {index + 1}"
+        try:
+            user, password, use_email = account.split(":")
+            use_email = use_email.upper()
+            ds = DSPACAPI(user, password, filename=f"DSPAC_{index + 1}.txt", creds_path="./creds/")
```
Comment (MaxxRK) [inline dspacAPI.py:39]: Nelson Dane also mentioned this, but the creds file should be saved as a .pkl or json.

**Comment by MaxxRK** [comment]
I used this today and I believe its ready to merge.

**Review by NelsonDane — APPROVED** [review]
LGTM

**Review by MaxxRK — APPROVED** [review]
Looks Good!

**Pull request merged**